Mathlib/Order/Filter/IsBounded.lean

```diff
@@ -396,19 +396,23 @@ end add_and_sum
 
 section mul
 
-lemma isBoundedUnder_le_mul_of_nonneg [Mul α] [Zero α] [Preorder α] [PosMulMono α]
-    [MulPosMono α] {f : Filter ι} {u v : ι → α} (h₁ : 0 ≤ᶠ[f] u)
-    (h₂ : IsBoundedUnder (fun x1 x2 ↦ x1 ≤ x2) f u)
-    (h₃ : 0 ≤ᶠ[f] v)
+lemma isBoundedUnder_le_mul_of_nonneg [LinearOrder α] [MulZeroClass α] [PosMulMono α]
```

Comment:
You are strengthening the typeclasses. Is your statement/proof still true under the weaker typeclasses?